content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -94,32 +112,43 @@ To encrypt the data:
          - identity: {}
    ```
 
-1. Set the `--encryption-provider-config` flag on the kube-apiserver to point to the location of the configuration file.
+1. Set the `--encryption-provider-config` flag on the kube-apiserver to point to
+   the location of the configuration file.
 1. Restart your API server.
 
+For details about the `EncryptionConfiguration` format, please check
+[API server encryption API reference](/docs/reference/config-api/apiserver-encryption.v1/).
```

Comment:
This is the sentence added. Other changes in this PR are all about markdown format.